services/lsst-condor.rst

```diff
@@ -1,7 +1,13 @@
+:orphan:
```

Comment:
I think this file should just be removed if it is orphaned; I don't see any value to keeping it around.

Reply:
We have current HTCondor users and testers that use this file for reference. I don't think we just want to yank it out from under them (at least not yet).